src/azure-blob-adapter/wsgi.py

```diff
@@ -0,0 +1,53 @@
+from azure.storage.blob import AppendBlobService
+from azure.common import AzureMissingResourceHttpError
+from dotenv import load_dotenv
+from logging import getLogger
+from os import environ
+from werkzeug.wrappers import PlainRequest, Response
+
+__all__ = ['application']
+
+load_dotenv()
+logger = getLogger(__name__)
+
+append_blob_service = AppendBlobService(connection_string=environ['AZURE_STORAGE_CONNECTION_STRING'])
+container_name = environ['AZURE_STORAGE_CONTAINER_NAME']
+
+
+@PlainRequest.application
+def application(request):
+    '''
+    :param PlainRequest request:
+    '''
+    try:
+        blob_name = request.headers['X-Tag']
+    except KeyError:
+        message = 'X-Tag header is required.'
+        logger.error(message)
+        return Response(message, status=400)
+
+    while True:
```

Comment:
I think using `while True` in production is very dangerous. If we want to retry, we'd better specify a retry limit, liket 3. Even a very large retry count is better than infinity.

Reply:
Clarified and removed while-loop